simulator/src/test/java/com/hedera/block/simulator/BlockStreamSimulatorTest.java

```diff
@@ -75,8 +80,12 @@ void setUp() throws IOException {
     }
 
     @AfterEach
-    void tearDown() {
-        blockStreamSimulator.stop();
+    void tearDown() throws InterruptedException {
+        try {
+            blockStreamSimulator.stop();
+        } catch (UnsupportedOperationException e) {
+            // ignore for now, temporary until consumer mode in simulator is added
```

Comment:
Maybe a good idea is to add a `@todo` with an issue number (create a new issue for that test) so we do not forget this. Based on #287.

Reply:
Changed 👍